hydromt_fiat/fiat.py

```diff
@@ -398,10 +406,13 @@ def setup_exposure_buildings(
                 max_potential_damage,
                 ground_floor_height,
                 extraction_method,
+                unit,
```

Comment:
unit should be delete from here and from the method

Reply:
its used in the function, why delete?

        if hasattr(length_unit, "value"):
            length_unit = length_unit.value

        self.exposure = ExposureVector(
            self.data_catalog,
            self.logger,
            self.region,
            length_unit=length_unit,
            damage_unit=damage_unit,
        )